### PR TITLE
Step name added in world route step to be used in Nav instructions

### PR DESCRIPTION
**Pull request by devfsmalik**
Step name added in iOS API WRLDRouteStep which is required for instruction formatting for Nav Widgets work.

### CODE REVIEW

File: src/public/WRLDRouteStep.h [hunk 1]
```diff
@@ -70,6 +70,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (CLLocationDistance) distance;
 
+/*!
+ @returns name of current step
```
Comment (ArslanJafri) [inline src/public/WRLDRouteStep.h]: Can this be updated to match rest of the comments. This is going to be showing up on our docs page (https://www.wrld3d.com/ios/latest/docs/api/Classes/WRLDRouteStep.html) so a better comment describing what this is, is extremely important.

Can you give me some details of what this variable contains. I think it contains the location name for this step but would like it confirmed before we decide on the docs comment.

**Review by ArslanJafri — CHANGES_REQUESTED** [review]

File: src/public/WRLDRouteStep.h (hunk 1 above)
Comment (devfsmalik) [reply src/public/WRLDRouteStep.h]: Yes, it contains step location name e.g "WRLD Office".

Comment (ArslanJafri) [reply src/public/WRLDRouteStep.h]: Can we update it to `@returns If available, the name of the location this step is on.`

Comment (devfsmalik) [reply src/public/WRLDRouteStep.h]: fixed

**Pull request merged**